xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVAAPI.h

```diff
@@ -22,9 +22,16 @@
 
 #include "system.h"
 
+#ifdef HAS_GL
 #include "cores/VideoPlayer/VideoRenderers/LinuxRendererGL.h"
+using CRendererVAAPIBase = CLinuxRendererGL;
```

Comment:
better write your own renderer for gles, having a this one dependent on gl and gles makes no sense